### PR TITLE
Upgrade YYeTs.app to v3.1.1

### PR DESCRIPTION
**Pull request by womeimingzi11**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Site looks inaccessible.

**Comment by womeimingzi11** [comment]
> Site looks inaccessible.

I have changed the homepage into right adress

File: Casks/yyets.rb [hunk 1]
```diff
@@ -1,13 +1,12 @@
 cask 'yyets' do
-  version '2.55'
-  sha256 '3ac6d420585003160ce3e4053c749502c8e034c2c63f34dcecb30b180e768e2d'
+  version '3.1.1'
+  sha256 'acc48e3a6c450149262fbdadea5350a176b8e8f79d1da4db282dd690fd8c9af3'
 
-  # appdown.rrys.tv was verified as official when first introduced to the cask
   url "http://appdown.rrys.tv/YYetsShare.#{version}.dmg"
-  appcast 'http://app.zimuzu.io/'
+  appcast 'http://app.rrys.tv/'
```
Comment (vitorgalvao) [inline Casks/yyets.rb]: `appcast` is not unnecessary. It is very much useful.

Comment (womeimingzi11) [reply Casks/yyets.rb]: I am so sorry about this mistake.

Comment (vitorgalvao) [reply Casks/yyets.rb]: Don’t worry about it. No harm done.

**Comment by vitorgalvao** [comment]
Thank you for the contribution and the fixes!

**Pull request merged**